charts/performancetesting/values.yaml

```diff
@@ -0,0 +1,20 @@
+
+replicaCount: 1 #This is not used for now
+namespace: dev
+schedule: "0 9 * * 1" #It runs At 9:00 every Monday
+image:
+  repository: quay.io/urbanos/perfomancetesting
+  pullPolicy: Always #This parameter is set "Always" due to the updates of the image
+  tag: development
+service:
+  ports: 8080
```

Comment:
Similar [comment about the port here](https://github.com/UrbanOS-Public/charts/pull/439/files#r1084647175) do we need a port open on the cronjob?